django_cache_manager/cache_manager.py

```diff
@@ -47,7 +47,7 @@ def iterator(self):
         except EmptyResultSet:
             return
         result_set = self.cache_backend.get(key)
-        if not result_set:
+        if result_set is None:
```

Comment:
consider changing to `if result_set is None or not result_set` to catch empty `[]`

Reply:
But doesn't empty [] mean that the cache was hit (successfully) but the result is empty? This is exactly the incorrect "miss" that I wanted to avoid.

Reply:
From the [Django docs](https://docs.djangoproject.com/en/1.11/topics/cache/):

> If the object doesn’t exist in the cache, cache.get() returns None

Reply:
I see, makes sense to me.